NetKAN/ThermonuclearTurbines.netkan

```diff
@@ -0,0 +1,23 @@
+{
+    "$kref": "#/ckan/spacedock/521",
+    "identifier": "ThermonuclearTurbines",
+    "spec_version": "v1.4",
+    "license": "CC-BY-NC-SA-4.0",
+    "depends": [
+        { "name": "CommunityResourcePack" },
+        { "name": "Firespitter" },
+        { "name": "SmokeScreen" },
+        { "name": "ModuleManager" },
+        { "name": "EskandareHeavyIndustries" }
+    ]
+    "install": [
+        {
+            "find": "Thermonuclear",
+            "install_to": "GameData/Eskandare_Heavy_Industries"
+        },
+        {
+            "find": "SPH",
```

Comment:
Sometimes I see comments from myself days earlier and can't for the life of me remember why I wrote it.